app/views/shared/_email_status.html.erb

```diff
@@ -28,7 +28,7 @@
                   </td>
                   <td>
                     <%- last_approver = index == @proposal.steps_in_list_order.count - 1 %>
-                    <%= status_icon_tag(display_status, @proposal.linear?, last_approver) %>
+                    <%= status_icon_tag(display_status, last_approver) %>
```

Comment:
this is what the email looks like for parallel approvals before this was removed

![screen shot 2015-12-16 at 10 13 12 am](https://cloud.githubusercontent.com/assets/601515/11849589/713d5fde-a3dd-11e5-9d02-fe8f85c62a14.png)